src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -133,6 +135,14 @@ export const GithubConfigurationGet = async (req: Request, res: Response, next:
 	gitHubAppId ? req.log.debug(`Displaying orgs that have GitHub Enterprise app ${gitHubAppId} installed.`)
 		: req.log.debug("Displaying orgs that have GitHub Cloud app installed.");
 
+	const gitHubProduct = gitHubAppId ? "server" : "cloud";
+
+	sendAnalytics(AnalyticsEventTypes.ScreenEvent, {
+		name: AnalyticsScreenEventsEnum.ConnectAnOrgScreenEventName,
+		jiraHost,
+		gitHubProduct
```

Comment:
This will help us determine how many customers (both cloud and server) are making it to the point where they can connect an org.